modules/fluid_properties/include/materials/SodiumPropertiesMaterial.h

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Material.h"
+
+#include "SodiumProperties.h"
+
+class SodiumPropertiesMaterial;
+
+template <>
+InputParameters validParams<SodiumPropertiesMaterial>();
+
+class SodiumPropertiesMaterial : public Material
+{
+public:
+  SodiumPropertiesMaterial(const InputParameters & parameters);
+
+protected:
+  virtual void computeQpProperties() override;
+
+  /// Coupled temperature variable
+  const VariableValue & _temperature;
+
+  /// Thermal conductivity
+  MaterialProperty<Real> & _k;
+
+  /// Enthalpy
+  MaterialProperty<Real> & _h;
+
+  /// Heat capacity
+  MaterialProperty<Real> & _cp;
+
+  /// Temperature from enthalpy
+  MaterialProperty<Real> & _t_from_h;
```

Comment:
I would suggest to use `T` for temperature to be consistent with the rest of the module. Also, little `t` is usually used for time in MOOSE - just to avoid confusion.